screens/store/checkout/CheckoutScreen.js

```diff
@@ -4,9 +4,22 @@ import { WebView } from "react-native-webview";
 import { SimpleContainer } from "../../../components/Container/SimpleContainer";
 import { LoadingIndicator } from "../../../components/LoadingIndicator/LoadingIndicator";
 import { styles } from "../../../styles/store/checkout/checkout-screen/styles";
+import Routes from "../../../routes/routes";
 
 export function CheckoutScreen({ navigation, route }) {
-  const { redirectSession } = route.params;
+  const { redirectSession, cameFrom } = route?.params || {};
+  if (!redirectSession) {
+    navigation.navigate(Routes.Cart);
```

Comment:
is it ok to call `navigate` inside the render function of a component? is there a case where the code actually gets here or is this more for developers not to make a mistake? (if it's the latter, I'd just throw an error I guess, saying a redirect session must be passed)